src/components/themed/ThemedButtons.js

```diff
@@ -169,7 +169,7 @@ function spacingStyles(props: Props, theme: Theme) {
 const getStyles = cacheStyles((theme: Theme) => {
   const commonButton = {
     alignItems: 'center',
-    borderRadius: theme.rem(1.5),
+    borderRadius: theme.rem(0.25),
```

Comment:
create new themed button for it, just for this PR change. then create second PR that removes this and changes borderRadius

Reply:
Ignore this comment.
Julia approved this change.